Article Main Page Method Bug Fix

Error: src/app/page-space-la/page-space-la.page.html:4:82 - error TS2554: Expected 0 arguments, but got 1.

4         <ion-button slot="start" class="ion-padding-start" (click)="exitEditMode(false)" *ngIf="editMode && this.authService.isAdmin()">
                                                                                   ~~~~~

  src/app/page-space-la/page-space-la.page.ts:10:16
    10   templateUrl: './page-space-la.page.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component PageSpaceLaPage.


Error: src/app/page-space-la/page-space-la.page.html:8:79 - error TS2554: Expected 0 arguments, but got 1.

8         <ion-button slot="end" class="ion-padding-end" (click)="enterEditMode(true)" *ngIf="!editMode && this.authService.isAdmin()">
                                                                                ~~~~

  src/app/page-space-la/page-space-la.page.ts:10:16
    10   templateUrl: './page-space-la.page.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component PageSpaceLaPage.


Error: src/app/page-space-la/page-space-la.page.html:78:69 - error TS2322: Type 'number' is not assignable to type 'string'.

78                     <app-article-main [contentCol]="articleCol[0]" [col]="1" [editMode]="editMode"></app-article-main>
                                                                       ~~~

  src/app/page-space-la/page-space-la.page.ts:10:16
    10   templateUrl: './page-space-la.page.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component PageSpaceLaPage.


Error: src/app/page-space-la/page-space-la.page.html:81:69 - error TS2322: Type 'number' is not assignable to type 'string'.

81                     <app-article-main [contentCol]="articleCol[1]" [col]="2" [editMode]="editMode"></app-article-main>
                                                                       ~~~

  src/app/page-space-la/page-space-la.page.ts:10:16
    10   templateUrl: './page-space-la.page.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component PageSpaceLaPage.


Error: src/app/page-space-la/page-space-la.page.html:84:69 - error TS2322: Type 'number' is not assignable to type 'string'.

84                     <app-article-main [contentCol]="articleCol[2]" [col]="3" [editMode]="editMode"></app-article-main>
                                                                       ~~~

  src/app/page-space-la/page-space-la.page.ts:10:16
    10   templateUrl: './page-space-la.page.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component PageSpaceLaPage.

--- a/Cyclops/src/app/page-space-la/page-space-la.page.html
+++ b/Cyclops/src/app/page-space-la/page-space-la.page.html
@@ -1,11 +1,11 @@
 <ion-header [translucent]="true">
     <ion-toolbar>
         <ion-title style="font-size: 150%;" *ngIf="!editMode">Articles</ion-title>
-        <ion-button slot="start" class="ion-padding-start" (click)="exitEditMode(false)" *ngIf="editMode && this.authService.isAdmin()">
+        <ion-button slot="start" class="ion-padding-start" (click)="exitEditMode()" *ngIf="editMode && this.authService.isAdmin()">
             <ion-icon name="chevron-back-outline"></ion-icon>
             <ion-label>&nbsp;&nbsp;Exit Edit</ion-label>
         </ion-button>
-        <ion-button slot="end" class="ion-padding-end" (click)="enterEditMode(true)" *ngIf="!editMode && this.authService.isAdmin()">
+        <ion-button slot="end" class="ion-padding-end" (click)="enterEditMode()" *ngIf="!editMode && this.authService.isAdmin()">
             <ion-icon name="settings-outline"></ion-icon>
             <ion-label>&nbsp;&nbsp;Manage Article Cards</ion-label>
         </ion-button>

--- a/Cyclops/src/app/page-space-la/page-space-la.page.html
+++ b/Cyclops/src/app/page-space-la/page-space-la.page.html
@@ -75,13 +75,13 @@
         <ng-container *ngIf="articleCol">
             <div [ngSwitch]="status1">
                 <div *ngSwitchCase="'Articles p1'">
-                    <app-article-main [contentCol]="articleCol[0]" [col]="1" [editMode]="editMode"></app-article-main>
+                    <app-article-main [contentCol]="articleCol[0]" [col]="'1'" [editMode]="editMode"></app-article-main>
                 </div>
                 <div *ngSwitchCase="'Articles p2'">
-                    <app-article-main [contentCol]="articleCol[1]" [col]="2" [editMode]="editMode"></app-article-main>
+                    <app-article-main [contentCol]="articleCol[1]" [col]="'2'" [editMode]="editMode"></app-article-main>
                 </div>
                 <div *ngSwitchCase="'Articles p3'">
-                    <app-article-main [contentCol]="articleCol[2]" [col]="3" [editMode]="editMode"></app-article-main>
+                    <app-article-main [contentCol]="articleCol[2]" [col]="'3'" [editMode]="editMode"></app-article-main>
                 </div>
             </div>
         </ng-container>

